src/tokenserver/secrets.py

```diff
@@ -0,0 +1,24 @@
+import binascii
+import hashlib
+from tokenlib import HKDF
+
+# Namespace prefix for HKDF "info" parameter.
+HKDF_INFO_NODE_SECRET = b"services.mozilla.com/mozsvc/v1/node_secret/"
+
+
+def derive_secrets(master_secrets, node):
```

Comment:
I wanted to use the mozsvc Python library to avoid having to implement this, but it doesn't support Python 3. This code comes from [this method](https://github.com/mozilla-services/mozservices/blob/fbb0910c0d23544d5cf30e258e33264a69e3ecad/mozsvc/secrets.py#L147) of the `DerivedSecrets` class that would have been used here.